Modules/_ctypes/stgdict.c

```diff
@@ -383,7 +383,7 @@ PyCStructUnionType_update_stginfo(PyObject *type, PyObject *fields, int isStruct
         size = 0;
         align = 0;
         union_size = 0;
-        total_align = forced_alignment;
+        total_align = forced_alignment != 0 ? forced_alignment : 1;
```

Comment:
```suggestion
        total_align = forced_alignment == 0 ? 1 : forced_alignment;
```

I think it's easier to read it that way but up to you.